tests/fixtures/qasm2/shared15.qasm

```diff
@@ -17,7 +18,7 @@ tdg q[3];
 rx(pi/4) q[0];
 ry(pi/2) q[1];
 rz(3*pi/4) q[2];
-p(pi/8) q[3];
+u3(0, 0, pi/8) q[3];
```

Comment:
Any reason for this update when `p` is already defined in `qelib1.inc` as 

```qasm
gate p(lambda) q { U(0,0,lambda) q; }
```

https://github.com/Qiskit/qiskit/blob/1.3.0/qiskit/qasm/libs/qelib1.inc#L24

Reply:
I think this update was before I added the `p` gate in `pyqasm`, will revert